File: GhostChat-iOS/GhostChat/Models/Room.swift
import Foundation

/// Состояние комнаты
struct Room {
    let id: String
    let isHost: Bool
    let createdAt: Date

    var isExpired: Bool {
        Date().timeIntervalSince(createdAt) > 10 * 60 // 10 минут TTL
    }
}

File: GhostChat-Android/app/src/main/java/com/ghost/chat/models/Room.kt
package com.ghost.chat.models

import java.util.Date

data class Room(
    val id: String,
    val isHost: Boolean,
    val createdAt: Date = Date()
) {
    /** Room TTL: 10 minutes */
    val isExpired: Boolean
        get() = (System.currentTimeMillis() - createdAt.time) > 10 * 60 * 1000
}
